### PR TITLE
test if libfulltext is pip installable

### PR DESCRIPTION
**Pull request by pseyfert**
As discussed in #42

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -9,3 +9,5 @@ before_install:
     - pip install pylint
 script:
     - pylint bin libfulltext
+after_success:
+    - pip install .
```
Comment (pseyfert) [inline .travis.yml]: I suspect `.` is not the best directory specification. Does any of you know if there is an environment variable in travis like `${CURRENT_CHECKOUT_DIR}`?

Comment (andrenarchy) [reply .travis.yml]: I think this is fine.

**Review by andrenarchy — CHANGES_REQUESTED** [review]
Actually I think `after_success` is not a good idea... then it doesn't fail if the `pip install .` fails, right? maybe we move it to `script`?

**Comment by pseyfert** [comment]
Hm, I would expect a CI fails if any of its steps fails. Let me try …

**Comment by andrenarchy** [comment]
From https://docs.travis-ci.com/user/customizing-the-build/#Breaking-the-Build:
> The exit code of after_success, after_failure, after_script and subsequent stages do not affect the build result. However, if one of these stages times out, the build is marked as a failure.

**Comment by pseyfert** [comment]
Indeed https://travis-ci.org/pseyfert/libfulltext/jobs/324081889

**Review by andrenarchy — APPROVED** [review]

**Comment by mfherbst** [comment]
We could directly upload the stuff to pip in `after_success`, right?

**Comment by pseyfert** [comment]
Yes, though let's discuss in #34 .

**Pull request merged**